craystack/codecs_test.py

```diff
@@ -41,23 +41,20 @@ def test_benford():
                                & ((1 << data_len) - 1)))
     check_codec(shape, cs.Benford64, data)
 
-
 def test_benford_inverse():
     shape = (2, 3, 5)
     data_len = rng.randint(31, 63, shape, dtype="uint64")
     data = ((1 << data_len) | (rng.randint(1 << 63, size=shape, dtype="uint64")
                                & ((1 << data_len) - 1)))
     check_codec(shape, cs.Benford64, data)
 
-
 def test_repeat():
     precision = 4
     n_data = 7
     shape = (2, 3, 5)
     data = rng.randint(precision, size=(n_data,) + shape, dtype="uint64")
```

Comment:
Always wondered if this should be 

```suggestion
    data = rng.randint(1 << precision, size=(n_data,) + shape, dtype="uint64")
```

Reply:
Yes, thanks!